runtime/autoload/dist/ft.vim

```diff
@@ -144,6 +144,14 @@ export def FTcfg()
   endif
 enddef
 
+export def FTcl()
+  if getline(1) =~ '/\*'
```

Comment:
Is checking only the first line enough?

Reply:
the file shoule have a header:

```
/*
 * XXX blahblah
 */
```

It should only occur in first line?